tests/CallbackTest.php

```diff
@@ -81,6 +81,9 @@ public function testViewUrlCallback(): void
         // simulate triggering
         $_GET[Callback::URL_QUERY_TRIGGER_PREFIX . $cb->name] = '1';
 
+        // Test for non sticky
+        $cb->setUrlTrigger($cb->name);
```

Comment:
I belive this is the reason why the original test was passing - why do we need the Behat test at all?

We can even extend the unit test to set $_GET with target name which should add the sticky trigger to the URL.

Reply:
> We can even extend the unit test to set $_GET with target name which should add the sticky trigger to the URL.

Not sure about this. If you think it will do it, then go ahead.

I thought that if stickyGet is set by means of another method, then the URL will be wrong and the test will still pass.
That is why I added the Behat test for it.